tests/lib.rs

```diff
@@ -71,6 +73,178 @@ fn it_works() {
 }
 
 #[test]
+fn it_propagates_writes_w_durability_sync_immediately() {
+    use distributary::{Base, BaseDurabilityLevel};
+
+    // set up graph
+    let mut g = distributary::Blender::new();
+    let (a, b, cq) = {
+        let mut mig = g.start_migration();
+        let a = mig.add_ingredient("a", &["a", "b"],
+                                   Base::new(vec![0], BaseDurabilityLevel::SyncImmediately).delete_log_on_drop());
```

Comment:
To make this look a bit neater, you may want to assign `BaseDurabilityLevel::SyncImmediately` to a variable with a short name instead :)